bindings/go/src/ucx/goucx.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2021, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+#include <ucp/api/ucp.h>
+
+extern void  ucxgo_completeGoSendRequest(void *request, ucs_status_t status, void *calback_id);
+
+extern void  ucxgo_completeGoTagRecvRequest(void *request, ucs_status_t status, ucp_tag_recv_info_t *info, void *calback_id);
+
+extern void  ucxgo_completeGoErrorHandler(ucp_ep_h ep, ucs_status_t status);
```

Comment:
why need 'extern'?

Reply:
https://pkg.go.dev/cmd/cgo#hdr-C_references_to_Go
> Go functions can be exported for use by C code in the following way:
```
//export MyFunction
func MyFunction(arg1, arg2 int, arg3 string) int64 {...}

//export MyFunction2
func MyFunction2(arg1, arg2 int, arg3 string) (int64, *C.char) {...}
```
They will be available in the C code as:
```
extern GoInt64 MyFunction(int arg1, int arg2, GoString arg3);
extern struct MyFunction2_return MyFunction2(int arg1, int arg2, GoString arg3);
```
It's a go functions that are visible in C. Cgo will have actual stub of these functions in it's internal files, like `_cgo_export.h`. So extern is just to tell the compiler that actual implementation would be in other files, but we need this header to access this functions in GO (set callback).

Reply:
ok